prow/config.yaml

```diff
@@ -136,7 +137,67 @@ presubmits:
         hostPath:
           path: /mnt/disks/ssd0
 
+  - name: auth-presubmit
```

Comment:
Does `name` need to be unique? How do we differentiate jobs on prow.istio.io?

Reply:
job name and script name are related. You can differentiate them in prow.istio.io